BUILDING.md

```diff
@@ -45,20 +41,36 @@ pngs_from_svg.py ic_audio.svg /path/to/Session/res/ 150 --color #000 --opacity 0
 pngs_from_svg.py ic_audio.svg /path/to/Session/res/ 150 --color #fff --opacity 1.00 --suffix _light
 ```
 
-Setting up a development environment
+Setting up a development environment and building from Android Studio
 ------------------------------------
 
 [Android Studio](https://developer.android.com/sdk/installing/studio.html) is the recommended development environment.
 
-1. Install Android Studio.
-2. Open Android Studio. On a new installation, the Quickstart panel will appear. If you have open projects, close them using "File > Close Project" to see the Quickstart panel.
-3. From the Quickstart panel, choose "Configure" then "SDK Manager".
-4. In the SDK Tools tab of the SDK Manager, make sure that the "Android Support Repository" is installed, and that the latest "Android SDK build-tools" are installed. Click "OK" to return to the Quickstart panel.
-5. From the Quickstart panel, choose "Checkout from Version Control" then "git".
-6. Paste the URL for the session-android project when prompted (https://github.com/loki-project/session-android.git).
-7. Android Studio should detect the presence of a project file and ask you whether to open it. Click "yes".
-9. Default config options should be good enough.
-9. Project initialisation and build should proceed.
+1. Open Android Studio. On a new installation, the Quickstart panel will appear. If you have open projects, close them using "File > Close Project" to see the Quickstart panel.
+2. From the Quickstart panel, choose "Checkout from Version Control" then "git".
+3. Paste the URL for the session-android project when prompted (https://github.com/loki-project/session-android.git).
+4. Android Studio should detect the presence of a project file and ask you whether to open it. Click "yes".
+5. Default config options should be good enough.
+6. Right now, we have kept the google-services.json file out of the public repository as it contains credentials for the firebase auth. We haven't yet made a way to build the app without this file or without firebase, as you don't technically need it to run the app. Email niels@loki.network to get this file for your build.
```

Comment:
For this you can also make a dummy firebase application and get the json file from there. Should we list that as an option?

Reply:
Yeah but I don't know how we'd do that, @nielsandriesse is seeing if it even matters if we just include the original file to the public or if there's anything sensitive